src/vfb_query_builder/query_roller.py

```diff
@@ -702,6 +711,12 @@ def anat_query(self, short_forms: List):
                                       self.anatomy_channel_image()],
                              pretty_print=True)
 
+    def anat_scRNAseq_query(self, short_forms: List):
+        return query_builder(query_short_forms=short_forms,
+                             query_labels=['Class', 'Anatomy'],
+                             clauses=[self.term(), self.anat_cluster_dataset()],
+                             pretty_print=True)
+
```

Comment:
Define an anatomy query consisting of a term clause and (defining the primary variable) and an anat_cluster_dataset clause.